Makefile

```diff
@@ -45,7 +45,7 @@ test:
 
 .PHONY: testacc
 testacc:
-	@go test -parallel=40 $(TEST) $(TESTARGS)
+	@export ACC_TEST_ENABLED=1 && go test -parallel=40 $(TEST) $(TESTARGS)
```

Comment:
Most all of the plugins use `VAULT_ACC` as the env var, but tbh, I think `ACC_TEST_ENABLED` is more clear. I don't think the inconsistency matters much in this case since there is a makefile target anyway 👍